src/data/move_animations.asm

```diff
@@ -0,0 +1,914 @@
+	dw MoveAnimation_5543
+	dw MoveAnimation_554a
+	dw MoveAnimation_5555
+	dw MoveAnimation_555e
+	dw MoveAnimation_556d
+	dw MoveAnimation_5574
+	dw MoveAnimation_557b
+	dw MoveAnimation_557e
+	dw MoveAnimation_5583
+	dw MoveAnimation_5583
+	dw MoveAnimation_5583
+	dw MoveAnimation_558c
+	dw MoveAnimation_5597
+	dw MoveAnimation_559c
+	dw MoveAnimation_55a1
+	dw MoveAnimation_55a4
+	dw MoveAnimation_55a9
+	dw MoveAnimation_55b4
+	dw MoveAnimation_55b4
+	dw MoveAnimation_55bf
+	dw MoveAnimation_55c4
+	dw MoveAnimation_55c9
+	dw MoveAnimation_55ce
+	dw MoveAnimation_55d5
+	dw MoveAnimation_55e0
+	dw MoveAnimation_55e5
+	dw MoveAnimation_55e6
+	dw MoveAnimation_55ed
+	dw MoveAnimation_55f2
+	dw MoveAnimation_55fb
+	dw MoveAnimation_55fe
+	dw MoveAnimation_5601
+	dw MoveAnimation_5604
+	dw MoveAnimation_5607
+	dw MoveAnimation_560a
+	dw MoveAnimation_560f
+	dw MoveAnimation_5612
+	dw MoveAnimation_561d
+	dw MoveAnimation_5628
+	dw MoveAnimation_562d
+	dw MoveAnimation_5632
+	dw MoveAnimation_5637
+	dw MoveAnimation_5644
+	dw MoveAnimation_564f
+	dw MoveAnimation_5654
+	dw MoveAnimation_5659
+	dw MoveAnimation_565e
+	dw MoveAnimation_5665
+	dw MoveAnimation_5668
+	dw MoveAnimation_5673
+	dw MoveAnimation_5673
+
+MoveAnimation_52c6:    ; (6:52c6)
+	ani_player 	 	   ANI_SPELL_MOVE
+	ani_opponent       ANI_GET_HIT
+	ani_normal 	 	   ANI_SHAKE1
+	ani_opponent       ANI_SHOW_DAMAGE
+	ani_end
+
+MoveAnimation_52cf:
+	ani_player 	 	   ANI_SPELL_MOVE
+	ani_opponent       $08
+	ani_normal 	 	   ANI_SHAKE2
+	ani_opponent       ANI_SHOW_DAMAGE
+	ani_end
+
+MoveAnimation_52d8:
+	ani_player 	 	   ANI_SPELL_MOVE
+	ani_opponent       ANI_THUNDER_SHOCK
+	ani_opponent       ANI_GET_HIT
+	ani_normal 	 	   ANI_SHAKE1
+	ani_opponent       ANI_SHOW_DAMAGE
+	ani_end
+
+MoveAnimation_52e3:
+	ani_player 	 	   ANI_SPELL_MOVE
+	ani_opponent       $0b
+	ani_opponent       $0c
+	ani_opponent       ANI_GET_HIT
+	ani_normal 	 	   ANI_SHAKE1
+	ani_opponent       ANI_SHOW_DAMAGE
+	ani_end
+
+MoveAnimation_52f0:
+	ani_player 	 	   ANI_SPELL_MOVE
+	ani_normal 	 	   $65
+	ani_normal 	 	   $0d
+	ani_opponent       ANI_GET_HIT
+	ani_normal 	 	   ANI_SHAKE1
+	ani_opponent       ANI_SHOW_DAMAGE
+	ani_end
+
+MoveAnimation_52fd:
+	ani_player 	 	   ANI_SPELL_MOVE
+	ani_opponent       $0e
+	ani_opponent       ANI_GET_HIT
+	ani_normal 	 	   ANI_SHAKE1
+	ani_opponent       ANI_SHOW_DAMAGE
+	ani_end
+
+MoveAnimation_5308:
+	ani_player 	 	   ANI_SPELL_MOVE
+	ani_opponent       $0f
+	ani_opponent       ANI_GET_HIT
+	ani_normal 	 	   ANI_SHAKE1
+	ani_opponent       ANI_SHOW_DAMAGE
+	ani_end
+
+MoveAnimation_5313:
+	ani_player 	 	   ANI_SPELL_MOVE
+	ani_normal 	 	   $10
+	ani_opponent       ANI_GET_HIT
+	ani_normal 	 	   ANI_SHAKE1
+	ani_opponent       ANI_SHOW_DAMAGE
+	ani_end
+
+MoveAnimation_531e:
+	ani_player 	 	   ANI_SPELL_MOVE
+	ani_opponent       $11
+	ani_opponent       ANI_GET_HIT
+	ani_normal 	 	   ANI_SHAKE1
+	ani_opponent       ANI_SHOW_DAMAGE
+	ani_end
+
+MoveAnimation_5329:
```

Comment:
There are a lot of places like right here where formatting is messed up. I think you have both tabs and spaces which doesn't render right

Reply:
Thanks!
I always forget it. ~_~
Gotta fix it.